### ISSUE TITLE
test of IterQueryInstances  in mock tests  fails

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#2749]
The test of InterQueryInstances fails now with an assertion:  Since this is a very low priority, I set the test to not run and documented the issue.  The ininiatial exception do to the querylanguage not gets an assertion as shown below.

```

E                           AssertionError: Missing ws-agnostic in-string on stderr in test:
E                           Verify instance command query execution. Returns error because mock does not support query
E                           Expected ws-agnostic in-string in any line:
E                           ------------
E                           Error: CIMError: 14 (CIM_ERR_QUERY_LANGUAGE_NOT_SUPPORTED): 
E                           ------------
E                           Actual output line(s):
E                           ------------
E                           Traceback (most recent call last):
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/pywbem/_cim_operations.py", line 6674, in IterQueryInstances
E                               pull_result = self.OpenQueryInstances(
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/pywbem/_cim_operations.py", line 8501, in OpenQueryInstances
E                               result = self._imethodcall(
E                             File "/usr/lib/python3.9/unittest/mock.py", line 1093, in __call__
E                               return self._mock_call(*args, **kwargs)
E                             File "/usr/lib/python3.9/unittest/mock.py", line 1097, in _mock_call
E                               return self._execute_mock_call(*args, **kwargs)
E                             File "/usr/lib/python3.9/unittest/mock.py", line 1158, in _execute_mock_call
E                               result = effect(*args, **kwargs)
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/pywbem_mock/_wbemconnection_mock.py", line 1188, in _mock_imethodcall
E                               result = methodnameattr(namespace, **params)
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/pywbem_mock/_wbemconnection_mock.py", line 1826, in _imeth_OpenQueryInstances
E                               context_tuple = self._mainprovider.OpenQueryInstances(
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/pywbem_mock/_mainprovider.py", line 3335, in OpenQueryInstances
E                               self._validate_open_params(FilterQueryLanguage, FilterQuery,
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/pywbem_mock/_mainprovider.py", line 2773, in _validate_open_params
E                               raise CIMError(
E                           pywbem._exceptions.CIMError: 14 (CIM_ERR_QUERY_LANGUAGE_NOT_SUPPORTED): FilterQueryLanguage 'DMTF:CQL' not supported
E                           
E                           During handling of the above exception, another exception occurred:
E                           
E                           Traceback (most recent call last):
E                             File "/home/kschopmeyer/virtualenvs/pycli39/bin/pywbemcli", line 33, in <module>
E                               sys.exit(load_entry_point('pywbemtools', 'console_scripts', 'pywbemcli')())
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/click/core.py", line 829, in __call__
E                               return self.main(*args, **kwargs)
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/click/core.py", line 782, in main
E                               rv = self.invoke(ctx)
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
E                               return _process_result(sub_ctx.command.invoke(sub_ctx))
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
E                               return _process_result(sub_ctx.command.invoke(sub_ctx))
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
E                               return ctx.invoke(self.callback, **ctx.params)
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/click/core.py", line 610, in invoke
E                               return callback(*args, **kwargs)
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/click/decorators.py", line 33, in new_func
E                               return f(get_current_context().obj, *args, **kwargs)
E                             File "/home/kschopmeyer/pywbem/pycli39/pywbemtools/pywbemtools/pywbemcli/_cmd_instance.py", line 565, in instance_query
E                               context.execute_cmd(lambda: cmd_instance_query(context, query, options))
E                             File "/home/kschopmeyer/pywbem/pycli39/pywbemtools/pywbemtools/pywbemcli/_context_obj.py", line 347, in execute_cmd
E                               cmd()  # The pywbemcli command function call.
E                             File "/home/kschopmeyer/pywbem/pycli39/pywbemtools/pywbemtools/pywbemcli/_cmd_instance.py", line 565, in <lambda>
E                               context.execute_cmd(lambda: cmd_instance_query(context, query, options))
E                             File "/home/kschopmeyer/pywbem/pycli39/pywbemtools/pywbemtools/pywbemcli/_cmd_instance.py", line 1370, in cmd_instance_query
E                               results = conn.PyWbemcliQueryInstances(
E                             File "/home/kschopmeyer/pywbem/pycli39/pywbemtools/pywbemtools/pywbemcli/_pywbemcli_operations.py", line 284, in PyWbemcliQueryInstances
E                               result = self.IterQueryInstances(
E                             File "/home/kschopmeyer/virtualenvs/pycli39/lib/python3.9/site-packages/pywbem/_cim_operations.py", line 6729, in IterQueryInstances
E                               assert self._use_query_pull_operations
E                           AssertionError
E                           
E                           ------------

tests/unit/pywbemcli/cli_test_extensions.py:392: AssertionError

Since this is in code that always returns an exception since the function was not implemented, I consider this as very low priortiy.

**Comment by andy-maier** [comment]
Fixed in the priority PR #2743 which is already merged. Please rebase whatever you ran to the latest master and re-run to verify it is solved.

**Comment by andy-maier** [comment]
Karl sated in the slack channel that the issue disappeared, as expected. Closing as resolved by another fix.
